hyperdbg/hprdbgctrl/code/debugger/commands/debugging-commands/r.cpp

```diff
@@ -366,5 +366,6 @@ CommandR(vector<string> SplittedCommand, string Command)
                 ShowMessages("err, invalid register\n");
             }
         }
+        delete RegD;
```

Comment:
Since size of the `DEBUGGEE_REGISTER_READ_DESCRIPTION` struct is so small, it's better to only have stack allocations instead of dynamic heap allocation.

@SinaKarvandi